docs: VS Code as default editor in Introduction to Eclipse Che (rhdevdocs-4580) (#2532)
codenvy/che-docs: modules/overview/pages/introduction-to-eclipse-che.adoc

* A multi-container workspace for each developer with the ability to replicate with a single click using {prod} factories.
* Pre-built stacks with the ability to create custom stacks for any language or runtime.
* An enterprise integration using OpenShift OAuth or Dex.

> * Browser-based IDEs; integration with link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code] and others such as link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community Edition].

Reviewer: We already have terminology for these IDE names in the docs:
https://www.eclipse.org/che/docs/stable/end-user-guide/selecting-a-workspace-ide/
So I suggest sticking with one common terminology for now.
Or, if you're not happy with reentering these manually, you can add new attributes for these in the upstream and downstream repos and then update the docs with the attributes.
If in doubt, you can double-check these IDE names with Nick on the forum channel.

```suggestion
* Browser-based IDEs; integration with link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code - Open Source] and others such as link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community Edition].
```

> * Browser-based IDEs; integration with link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code - Open Source IDE] and other web IDEs such as link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community IDE].

Reviewer 2: ```suggestion
* Browser-based IDEs; integration with link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code] and others such as link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains IntelliJ IDEA Community Edition].
```

* Support of tools protocols, such as the Language Server Protocol or Debug Adapter Protocol.
* A plugin mechanism compatible with Visual Studio Code extensions.

> * A software development kit (SDK) for creating custom cloud developer platforms.

Reviewer: ```suggestion
* A software development kit (SDK) for creating custom cloud developer platforms.

```
